ludwig/features/numerical_feature.py

```diff
@@ -81,17 +84,17 @@ def add_feature_data(
         dataset[feature[PROC_COLUMN]] = dataset_df[feature[COLUMN]].astype(
             np.float32).values
         if preprocessing_parameters['normalization'] is not None:
-            if preprocessing_parameters['normalization'] == 'zscore':
-                mean = metadata[feature[NAME]]['mean']
-                std = metadata[feature[NAME]]['std']
-                dataset[feature[PROC_COLUMN]] = (dataset[
-                                                     feature[
-                                                         PROC_COLUMN]] - mean) / std
-            elif preprocessing_parameters['normalization'] == 'minmax':
-                min_ = metadata[feature[NAME]]['min']
-                max_ = metadata[feature[NAME]]['max']
-                values = dataset[feature[PROC_COLUMN]]
-                dataset[feature[PROC_COLUMN]] = (values - min_) / (max_ - min_)
+            normalization_type = preprocessing_parameters['normalization']
+            NumericTransformer = get_from_registry(
+                normalization_type,
+                numeric_transformation_registry
+            )
+
+            numeric_transformer = NumericTransformer(**metadata[feature[NAME]])
+
+            values = dataset[feature[PROC_COLUMN]]
+            dataset[feature[PROC_COLUMN]] = numeric_transformer.transform(
+                values)
```

Comment:
as we are not using `values` anywhere else, this can become a single line:
```
dataset[feature[PROC_COLUMN]] = numeric_transformer.transform(dataset[feature[PROC_COLUMN]])
```

Reply:
done